### PR TITLE
[executor]:support dispatch task

### PR DESCRIPTION
**Pull request by hanfei1991**
Support grpc interface to accepte dispatch worker

### CODE REVIEW

**Comment by amyangfei** [comment]
ref task-2 in #93

File: lib/worker.go [hunk 1]
```diff
@@ -79,6 +79,18 @@ func NewBaseWorker(
...
 }
 
+func (w *BaseWorker) ID() WorkerID {
+	return w.id
+}
+
+func (w *BaseWorker) Workload() model.RescUnit {
+	wl, err := w.impl.Workload()
+	if err != nil {
+		panic(err)
```
Comment (amyangfei) [inline lib/worker.go:89]: Better to use `log.L().Panic`.
Besides we may add some error passing mechanism in the feature, single worker error should not affect the whole runtime.

File: executor/server.go [hunk 2]
```diff
@@ -200,6 +226,18 @@ func (s *Server) Run(ctx context.Context) error {
 		return nil
 	})
 
+	pollCon := s.cfg.PollConcurrency
+	if pollCon == 0 {
+		pollCon = sysruntime.NumCPU()
+	}
```
Comment (amyangfei) [inline executor/server.go:232]: We can put this logic to `Config.Parse` function in executor/config.go

File: executor/server.go [hunk 3]
```diff
@@ -129,8 +137,15 @@ func (s *Server) ResumeBatchTasks(ctx context.Context, req *pb.PauseBatchTasksRe
 	return &pb.PauseBatchTasksResponse{}, nil
 }
 
-func (s *Server) DispatchTask(ctx context.Context, request *pb.DispatchTaskRequest) (*pb.DispatchTaskResponse, error) {
-	panic("implement me")
+func (s *Server) DispatchTask(ctx context.Context, req *pb.DispatchTaskRequest) (*pb.DispatchTaskResponse, error) {
+	log.L().Info("dispatch task", zap.String("req", req.String()))
+	workerID := "dummy"
```
Comment (nongfushanquan) [inline executor/server.go:142]: why  a fixed id?

**Review by amyangfei — APPROVED** [review]

File: lib/worker.go [hunk 4]
```diff
@@ -83,6 +83,18 @@ func NewBaseWorker(
 	}
 }
 
+func (w *BaseWorker) ID() WorkerID {
+	return w.id
+}
+
+func (w *BaseWorker) Workload() model.RescUnit {
+	wl, err := w.impl.Workload()
```
Comment (amyangfei) [inline lib/worker.go:91]:
```suggestion
	wl, err := w.Impl.Workload()
```

**Pull request merged**